tyr/security_groups/chef_nodes.py

```diff
@@ -0,0 +1,11 @@
+rule = 'chef-nodes'
+
+definition = {
+    'rules': [
+        {
+            'port': 22,
+            'source': 'p-ops-vpn'
```

Comment:
You're right, I was just looking at the rules in `chef-nodes` and I didn't look for duplicate rules. I'll remove that.